tests/test_cli/test_add/test_skill.py

```diff
@@ -77,16 +78,7 @@ def setup_class(cls):
 
         os.chdir(cls.t)
         result = cls.runner.invoke(
-            cli,
```

Comment:
The tests should be run for the different registry types so we know they all work. You can mock the absence of the remote registry?